Rakefile

```diff
@@ -2,11 +2,9 @@
 
 require "bundler/gem_tasks"
 require "rspec/core/rake_task"
-
-RSpec::Core::RakeTask.new(:spec)
-
 require "rubocop/rake_task"
 
+RSpec::Core::RakeTask.new(:spec)
```

Comment:
Just moving things around so all the `require` statements and all the task definitions are grouped together